app/domain/healthchecks/monthly_aggregations.rb

```diff
@@ -0,0 +1,23 @@
+module Healthchecks
+  class MonthlyAggregations
+    include Concerns::Deactivable
+
+    def name
+      :aggregations
+    end
+
+    def status
+      aggregations.positive? ? :ok : :critical
+    end
+
+    def message
+      "ETL :: no aggregations of metrics for yesterday" if status == :critical
```

Comment:
Could you please change the message to `...no monthly aggregations...`

Reply:
Thanks @pmanrubia - [fixed](https://github.com/alphagov/content-performance-manager/pull/1153/commits/5f2a433ed28162d505c9dc798180201753d2351d)